src/Card/index.jsx

```diff
@@ -24,7 +24,7 @@ const Card = React.forwardRef(({
         clickable: isClickable,
       })}
       ref={ref}
-      tabindex={isClickable ? '0' : '-1'}
```

Comment:
[inform] This was fixed in a separate PR and should be all set upon a rebase of this branch.

Reply:
Yeah, rebasing removed it, thanks!